overlay.d/05rhcos/usr/libexec/rhcos-rebuild-selinux-policy

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+RHEL_VERSION=$(. /usr/lib/os-release && echo $RHEL_VERSION)
+echo -n "RHEL_VERSION=${RHEL_VERSION}"
+case "${RHEL_VERSION}" in
+  8.[0-6]) echo "Checking for policy recompilation";;
+  *) echo "Assuming we have new enough ostree"; exit 0;;
+esac
+
+ls -al /{usr/,}etc/selinux/targeted/policy/policy.31
```

Comment:
I'm not 100% sure this is stable across all of our releases.

Reply:
Just above this, we exit if the OS isn't RHEL 8.[0-6].  This code very intentionally will not run on RHEL9, because we don't need it there.